packages/s/selecto.json

```
@@ -0,0 +1,36 @@
+{
+  "description": "Selecto.js is a component that allows you to select elements in the drag area using the mouse or touch.",
+  "filename": "selecto.min.js",
+  "homepage": "https://daybrush.com/selecto",
+  "keywords": [
+    "select",
+    "selecto",
+    "selection",
+    "selectable",
+    "moveable"
+  ],
+  "license": "MIT",
+  "name": "selecto",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/daybrush/selecto.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "selecto",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js?(.map)"
```
Open wildcards are dangerous, let's just match what we need. [Glob test](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.js%3F%28.map%29&matches=false&tests=%2F%2F%20Imported%20from%20selecto%20NPM%20package&tests=.editorconfig&tests=LICENSE&tests=rollup.config.js&tests=dist%2Fselecto.cjs.js&tests=dist%2Fselecto.esm.js&tests=dist%2Fselecto.js&tests=dist%2Fselecto.min.js&tests=jsdoc.json&tests=package.json&tests=tsconfig.declaration.json&tests=tsconfig.json&tests=tsconfig.test.json&tests=tslint.json&tests=dist%2Fselecto.cjs.js.map&tests=dist%2Fselecto.esm.js.map&tests=dist%2Fselecto.js.map&tests=dist%2Fselecto.min.js.map&tests=.github%2FISSUE_TEMPLATE.md&tests=README.md&tests=declaration%2Fconsts.d.ts&tests=src%2Fconsts.ts&tests=declaration%2Findex.d.ts&tests=src%2Findex.ts&tests=declaration%2Findex.umd.d.ts&tests=src%2Findex.umd.ts&tests=declaration%2FSelecto.d.ts&tests=declaration%2Ftypes.d.ts&tests=src%2Ftypes.ts&tests=declaration%2Futils.d.ts&tests=src%2Futils.ts&tests=src%2FSelecto.tsx)